src/main/resources/gui/index.html

```diff
@@ -57,11 +59,29 @@
 
     fetch('/', requestOptions)
       .then(response => response.text())
-      .then(result => output.innerText = result)
+      .then(result => {
+        output.innerHTML = result;
```

Comment:
I am not a javascript expert, but why change from innerText to innerHTML here? Doesn't it work with innerText? I mean with innerText security should be high

Reply:
Well the problem with innerText is that the result won't actually be displayed on the result page. Its only showing the text which than you will need to put in a html file to actually see the result of the validation. I dont know if this is wanted but for a 0815 user this would be easier to validate their XRechnungen.

Reply:
What I mean by that is:

With innerHTML you see this as result:
![image](https://user-images.githubusercontent.com/5787100/108352586-5a38fb80-71e7-11eb-98d4-6c4b4c1e321d.png)

With innerText you see this as result:
![image](https://user-images.githubusercontent.com/5787100/108352675-789ef700-71e7-11eb-99ec-75d582f9567e.png)

Reply:
Well, then I want definetely innerText! The report isn't html, only can contain html. If you want to display the html parts, we need a different solution and I am not sure whether this is in scope of the validator gui.

Reply:
Changed back to innerText. Now the report data gets extracted from the div-box like you required it.